homeassistant/components/proxmoxve/strings.json

```diff
@@ -0,0 +1,26 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "ProxmoxVE",
+        "description": "Please fill out the form and then press submit when you're ready",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "realm": "Realm",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
+        }
+      }
+    },
+    "error": {
+      "auth_error": "[%key:common::config_flow::error::invalid_auth%]",
+      "ssl_rejection": "Could not verify the SSL certificate",
```

Comment:
should this be changed to a translation lookup like the others?

Reply:
I didn't find common translation key for SSL certificate error.